src/Umbraco.Web.BackOffice/Controllers/TrackedReferencesController.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Models.ContentEditing;
+using Umbraco.Cms.Core.Models.Entities;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Web.BackOffice.ModelBinders;
+using Umbraco.Cms.Web.Common.Attributes;
+using Umbraco.Cms.Web.Common.Authorization;
+
+namespace Umbraco.Cms.Web.BackOffice.Controllers
+{
+    [PluginController(Constants.Web.Mvc.BackOfficeApiArea)]
+    [Authorize(Policy = AuthorizationPolicies.SectionAccessMedia)]
+    [Authorize(Policy = AuthorizationPolicies.SectionAccessContent)]
```

Comment:
I think we will need an "OR" policy here instead of this, otherwise you will have forbidden-errors when you only have access e.g to content

```suggestion
    [Authorize(Policy = AuthorizationPolicies.SectionAccessContentOrMedia)]
```